libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -50,7 +50,13 @@ const AP_Param::GroupInfo AC_PrecLand::var_info[] = {
     // @Units: Centimeters
     AP_GROUPINFO("LAND_OFS_Y",    4, AC_PrecLand, _land_ofs_cm_y, 0),
 
-    // 5 RESERVED for EKF_TYPE
+    // @Param: EST_TYPE
+    // @DisplayName: Precision Land Estimator Type
+    // @Description: Specifies the estimation method used.
+    // @Values: 0:RAW_MEASUREMENTS, 1:TWO_STATE_KF_PER_AXIS
```

Comment:
Is it possible to get more info on which method would be better with what hardware ?